floodviz/static/js/map.js

```diff
@@ -100,45 +99,66 @@
 			add_paths(options.rivers_data, "river");
 			add_circles(options.ref_data, "ref-point", 2);
 			// Add sites and bind events for site hovers
-			var sites = add_circles(options.site_data, "gage-point", 3);
+			var sites = add_circles(options.site_data, "gage-point", 3, 'id');
 			sites.selectAll("circle")
-				.on('mousemove', function(d) {return self.mousemove(d.properties.name, d.properties.id)})
-				.on("mouseout", function() {return self.mouseout()});
+				.on('mousemove', function (d) {return self.mousemove(d.properties.name, d.properties.id)})
+				.on("mouseout", function (d) {return self.mouseout(d.properties.id)})
+				.on('click', function (d) { return self.click(d.properties.id)});
 			// Debug points
 			if (FV.mapinfo.debug) {
 				add_circles(options.bounds, "debug-point", 3)
 			}
 		};
+
 		/**
-		 * Handle all mouse over movements for calling element. 
+		 * Handle all mouse over movements for calling element.
 		 */
 		self.mousemove = function (sitename, sitekey) {
-			var gage_point_cords = document.getElementById('map'+sitekey).getBoundingClientRect();
+			FV.hydro_figure.activate_line(sitekey);
```

Comment:
You should talk with Garritt. We want to avoid using global variables in this module patterns. You are tying all potential map figures to this specific hydro_figure. Also this function does not necessarily need to be applied to mousemove. It affects a change on the map visualization and the function name should reflect that.